keys.go

```diff
@@ -32,6 +30,19 @@ type ssbSecret struct {
 	Public  string   `json:"public"`
 }
 
+func SecretPerms() os.FileMode {
+	// SecretPerms are the file permissions for holding SSB secrets.
+	var perms os.FileMode
+
+	switch runtime.GOOS {
+	case "windows":
+		perms = 0666
```

Comment:
A person reading the source code will question why the permissions are made more lenient under windows.  Adding a comment about why will help them.